suite/tests/CMakeLists.txt

```diff
@@ -3013,6 +3013,10 @@ else (UNIX)
   tobuild(security-win32.except-guard security-win32/except-guard.c)
   tobuild(security-win32.thread-setcontext security-win32/thread-setcontext.c)
   tobuild(security-win32.singlestep security-win32/singlestep.c)
+  if (NOT X64)
+    # no way to set debug register in 64 bit
+    tobuild(security-win32.except-debugreg security-win32/except-debugreg.c)
+  endif (NOT X64)
```

Comment:
so all the code above was for 32 bit only, but most code does not have the conditional macro except the one that checks debug register value.
Why not let that one return false on 64-bit and add comment to explain why we did not do x64. It seems more consistent.